mantidproject/mantid
# Fixed the bugs in muon analysis from issue 19613

Fixed two bugs from muon analysis GUI. 

**To test:**
There are two tests:

1. Load some data and then go into the data analysis tab and add a function. Do a fit. Then go to the home tab and change the data set. Then go back to the data analysis tab, the function should still be present. Go to the settings tab and change the state of the multiple fitting checkbox. Go to the data analysis tab and the GUI should look different and the function should be gone. 

2. With multiple fitting selected (in settings tab) the fit menu should not include "sequential fit" or "evaluate". In all other modes these two options should be present.  

Fixes #19613 


These are bugs that were caused by development for this release. So does not need to be in the release notes. 
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
